### ISSUE TITLE
Installation Process - Change/Add default folders, Update variable %PATH%

### ISSUE THREAD

**Issue by rompik** [eryar/PipeCAD#44]
During installation process, system offer to install software to D:\PipeCAD-1.0.XX. 
![Installation Folder](https://user-images.githubusercontent.com/12736532/171720711-5b6d940e-f422-4246-a867-c1114573aaf0.png)

I think because of you have drive D: as HDD and use it by default. But for common users, D drive can be DWD drive or can be not exists at all. Also, sometimes companies have strong IT security rules for installation software only to **%programfiles%**. I think will be reasonable to use folder **C:\Program Files\PipeCAD** (without version of software) by default, as it used by other software. This is not an obligation as it's possible to select required folder during installation process, but can simplify a little installation process for common users.

**Comment by rompik** [comment]
After testing installation to %programfiles% I found more headache than benefits: need to update all paths in batch files, PipeCAD can't open projects (even moving Project to folder **C:\Data\Projects** where I can write files w/out Local Admin rights).  It could be reasonable only to install to folder C:\PipeCAD as C: drive is usual for many users.

**Comment by rompik** [comment]
Also, I think will be good for future using of PipeCAD in different companies to allow customizing default folders during installation process: for example, Projects folders and Python libraries folder can be installed in different places like in PDMS Projects and pmllib. But by defaults, all folders will be placed in installation folder. I made possible screenshot of such customization: 
![Selection of path for Projects](https://user-images.githubusercontent.com/12736532/171736580-922605af-bcde-4bd2-8f97-4b0c94f5a1d1.png)

**Comment by eryar** [comment]
Yes, install to C:\Program Files\PipeCAD may have local admin rights problem.
Ok, I will change the setup to support set these default path like PDMS.

**Comment by eryar** [comment]
> After testing installation to %programfiles% I found more headache than benefits: need to update all paths in batch files, PipeCAD can't open projects (even moving Project to folder **C:\Data\Projects** where I can write files w/out Local Admin rights). It could be reasonable only to install to folder C:\PipeCAD as C: drive is usual for many users.

`C:\Program Files\PipeCAD` maybe the space in path in bat file will cause problem.

**Comment by rompik** [comment]
I fixed white spaces in all bat files, and even moved Project folder to other directory w/out white spaces, but looks like problem is deeper. In any case now it's not important issue to fix it.

**Comment by rompik** [comment]
Also, please, can you add checkbox to add folders **%PIPECAD_EXE%** and **%PIPECAD_EXE%\Scripts** to system variable **%PATH%**. It will allow updating pip and install python packages w/out doing additional steps to add these folders manually to **%PATH%**. I found an example of such a checkbox during the installation process.
![image](https://user-images.githubusercontent.com/12736532/171853473-a019cbd2-0af0-4b72-91ae-fd337998710d.png)
Without it, if I don't have Python installation on my machine, I will have errors during using of python.

**Comment by eryar** [comment]
OK, I will check this for setup program.

**Comment by eryar** [comment]
1. Select project and python lib path:
![image](https://user-images.githubusercontent.com/8439181/173183414-9ad93d3d-19fb-455d-861b-1fc3421f3328.png)

2. Add folders %PIPECAD_EXE% and %PIPECAD_EXE%\Scripts to system variable %PATH% 
![image](https://user-images.githubusercontent.com/8439181/173183427-49796263-3179-4949-be49-18ef233367af.png)

**Comment by rompik** [comment]
I tested setup of separated Projects and Libraries environment during PipeCAD installation (ver. 1.0.14), and it works good. There is only one issues which I discovered after. During of new project creation, PipeCAD added new evars.bat in **%PROJECTS_DIR%** folder, where wrote path to new project evars. As I understood, PipeCAD had to update **evarsProjects.bat** file to append a new project path and does not create extra **evars.bat**. 
![Extra evars bat and missed path to new project](https://user-images.githubusercontent.com/12736532/174023644-33f74553-283f-408a-bab4-36251fca8165.png)

Manual correction of evarsProjects.bat is not big deal, but I think better to update code for project creation to avoid this small inconvenience.
